packages/financial-templates-lib/src/price-feed/BigQueryPriceFeed.js

```diff
@@ -0,0 +1,155 @@
+const { PriceFeedInterface } = require("./PriceFeedInterface");
+const { BigQuery } = require("@google-cloud/bigquery");
+const moment = require("moment");
+const highland = require("highland");
+const { parseFixed } = require("@ethersproject/bignumber");
+const { createQuery } = require("../queries/GasEthQuery");
+
+const client = new BigQuery();
+
+// An implementation of PriceFeedInterface that uses BigQuery to retrieve prices.
+class BigQueryPriceFeed extends PriceFeedInterface {
```

Comment:
High-level comment: do we have a strategy for testing this in any way? Not necessarily just concerned about the query itself, but also making sure that this class is bug-free.

Reply:
I'm working on writing a test similar to the tests for the other price feeds. Should be in shortly!